Sources/Exporters/OpenTelemetryProtocolHttp/logs/OtlpHttpLogExporter.swift

```diff
@@ -63,16 +64,16 @@ public class OtlpHttpLogExporter: OtlpHttpExporterBase, LogRecordExporter {
       switch result {
       case .success:
         self?.exporterMetrics?.addSuccess(value: sendingLogRecords.count)
+        completion?(.success)
       case let .failure(error):
         self?.exporterMetrics?.addFailed(value: sendingLogRecords.count)
         self?.exporterLock.withLockVoid {
           self?.pendingLogRecords.append(contentsOf: sendingLogRecords)
         }
         OpenTelemetry.instance.feedbackHandler?(error.localizedDescription)
+        completion?(.failure)
       }
     }
-
-    return .success
```

Comment:
this blind `return .success` even when the export failed was why persistence didn't work out of the box